hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -50,16 +51,21 @@
   private static final int SECONDS_FROM_MILLIS = 1000;
 
   private final HdfsFrameworkConfig hdfsFrameworkConfig;
+  private final HdfsMesosConstraints hdfsMesosConstraints;
   private final LiveState liveState;
   private final IPersistentStateStore persistenceStore;
+
   private final DnsResolver dnsResolver;
   private final Reconciler reconciler;
 
+
   @Inject
   public HdfsScheduler(HdfsFrameworkConfig hdfsFrameworkConfig,
     LiveState liveState, IPersistentStateStore persistenceStore) {
 
     this.hdfsFrameworkConfig = hdfsFrameworkConfig;
```

Comment:
to be consistent with existing style, it should be 
this.hdfsMesosConstraints 
+           = new HdfsMesosConstraints(hdfsFrameworkConfig); We should pass the config value to HdfsMesosConstraints in case the config is updated at runtime in which case we can query the latest version.